eng/CredScanSuppression.json

```diff
@@ -26,8 +26,6 @@
ps4UvaYbCyR1leuT7bOWAbXn7Ztet46Jz4WKun5NyKIXtWpkO8sgsy20uuQkqSjWVuOKa6nCRkZ5iY/kb3AkWRqK7snBt994A21kKG8fZU670Dw1s3tixtLlE2WDwYnb3fULJzmtTXYkyaRFbWoFJPHrPMu9K9ZQTY/WXTcFSj2qgQUl05SQOBiJo7bn1kVT+XTTrCP5OELO0vvaIUV9omMwua9oUegWgS0GkADhvc7g2PkJw5ugoW23MFrLPU7Jt2MLXnBF3dhu6xkf7NMGCa0330On2oAy4ru3jsIqKkrxwz26uRrDz4Zy2G3lUbcPaP7T7IYdu44E/139dNxHZvoGuRFqWISbj5nkY5pcQUrgGZOpOe9ygmOSeUVY7dIJxFKW7X1O3Nl4LpoaKW42nTM7xuoVQ4O/GgapHqvgyKjiSzL9zrRZAXEprKQAQMCbreYmuzeJoXOugqYE9wHpEnePWpK7GpA+0wAJ3gsqFEaLS6PAUXNapX5af9VPrZenBBYFUIxPUn4uJz+p2PGMWA90H4OXv4eTlQ2lqqEqDYZ5MuwtIwt6vH1FO7h2GqiMbVN/ckTSpkFKaUbsiqESKfzD712YyNKmsKKU09e2c3SagCCW8zIj7kgsraDcKuvqOqjnMSXlxo+MgNcaYySHAzNDxL94Wy7sOIGogZmTEDEVdPvmdZQ5TKtan3YwvsUx5mtxU9t1uQIilK7PIbf1RrQimq6UXpQkatc1r1t+Ngwy2ZniZzwakuVc4DkWq0eaDHjR2S5a1WZtAc8UIHM/EQNcVZaQejBjR7QJmPY5eTgmE+7IbQMbn88nNdElKipmlqS3c6wO57L0zRSvbrBCxUuVI4BzdVfREuxm+Kev1paAS44rcygRzsRGr9gEoee/I6VgNbGjCV08H8UiF5K0/VbN93+EtaefNQBaU0fFLZUf1nxMgtgTJKqU2ku/d6R82B/4LWCDnvXABVCQ+1WmB5jXUzwYAFVXHqpZbGdE7qK663GRvT0CIN2FwljZMTtYNXCQO6TlseVLX8CrLdxGioJol85NWrj3YSD//gG3Ds0RiH+0fXcuN67in2RC3WKfaUCcUTur71Tau2f37SWZ9FrzOHQqJVoP/dKTG1UJ8wggVBBgkqhkiG9w0BBwGgggUyBIIFLjCCBSowggUmBgsqhkiG9w0BDAoBAqCCBO4wggTqMBwGCiqGSIb3DQEMAQMwDgQIAMlOFXUURPECAggABIIEyFYuslCPqytr0m3AvBpTJMm3Umv94Xg6z+lhl6oQ/ClfV7TBaUb1gvM12z7+hSbuyZfFJwm/rK5c28BHVpZL5zvehnhRALjt67yIEVhMmnqeSv9PIR7F1rzrGBGXle3i2MCzDQD3+QRD/m3HUF/WD9LLx9Q0ucUfdz4i0q07IbRAC1DP/xkmA2uqi/zWWIdec31tdqbyR7tA2hUyXXpVG5ZxxyHuwlEFZEcHpsHnDU405ih45+WhcA+R7CM6bwzLsRf1TZ7Ljx3Fw848G0Q6k4OvOmRyWtCAP/soJCPvuOXnIvJXOJNkDtv5FyoQF+xya9gfb6a+t/kbwHFhM14t6ryCZ525f86PEizcfijTDvdE2E1EqFZlbnBVCHwJ9eJucjSxXmpgslbdroNHZpMNA+bDZiqgcgcOHPK+MaE7KEoiRKkJ7Q4T3XWfrLqDaAtFmhINJw7HmSueliyOEM9HvVHa8CKA1IM8xdF/nrsGEYrw/GUtPaF1ZXEBoeFO1yH+6JkT2SyGbzei2KACShquZpFqJvH6gStxZP5oaOT0UIE2EKz9+2peTahE5XL3L7Xmq/UqezsRKOBBsR+uTSB6w8dWmweaaloBML1iRsNJLDFtZPJvi7zWMq0xjUxoL3ulMJ8jP2/u1mum0lylO26IFqg7V744gSt1FPHR62zwG2zV2b5hu1/hNnhXeE9KMsdwwdfLUkcqXrNBBqkDjF12S4rt7HCaAjx1fE/O/oMqC8YEO3TYa7mL4maj7lR9Ja0n7gVHpKGIy6IUevadPhxcvM1/dMaxO/hd4xCDQC1V8BYqxsv6QH4dbxOj3YTuxD7mRUSI7m4gQbG2jDlUYTWMdCaQow2RCPBE7JWk7efvX/RwknQXi26TcGsVhAZ0/5Tf2+KP0pOBZ5Y+4sQcbSBIXTQqTR4+qTEmTosSj4OTDOO6bS+bVnmsaNAJ9OVa9bNAXSOl+6WPEjNR3z42W/fd+sUk0Y0GU4YcNiBIfMPkkQksVmqU3Y0yyuazrZkk/32kqSrONfxV9JbkgLGwS4O2LBDCubxHyBoakqLvjyFie/jYp7Hu4R6XEfD0mgpho6hhWHh7gy54yA/RiKoZZ7IyJv14KbFzs0f8PEbSx75RpfVEtQD+RZyIby7ompwRnHLvw04gj0kWy1MR84QUIL7yoJJIOgL6Kcgci4M5ZaVdcNAr0CdmjNKfOB7DIvDSxClZjL94zObKxsnExamxs593HLkKJkEdhskpPmaqL/Tt3IaLqxuJsFz7iI2KPRwD9N9mfE9Sfc9yvJiL53Q3sZq9j3oq1bUYa1ubbv0+4Fj3Swg4FmNz7TCN7swtb2BRnwTLh9CRN3BgWaRPIOxmCeosl1GzqWv+sN/GbKeJ8Nylmk1NzaVeJg0NkK2T9hlstVfmdRm+Xk4V30pG/FD67251T1vkLCHP6DJysoEGsfiS2OM+/x3onMvrB6T+hfUmtvxKcnQ7d4DMfmHC9wj/YZkULLYIa+sd9Pbr1U2mJ6lqyWsPUM7XLDMfzN1XOgIDuzcYxTRR5TDiYyg9hYksMNeqSisCsy/ODtMKwk73m2+0GdvIGbPQfV7cgGBEBzN9Wh2BLWtsZq107ZerM02XmRFmq/MX0TaOsmVXhTElMCMGCSqGSIb3DQEJFTEWBBTsd3zCMw1XrWC/MBjgt8IbFbCL8jAxMCEwCQYFKw4DAhoFAAQULqAXFUm78ZQ7CCRuYcS3O9qq21EECJq6mOwqNtpgAgIIAA==",
```

Comment:
@scbedd is this what was causing the RegEx to be too large? If so I also agree we should remove this as it shouldn't really be in the placeholders anyway.